apps/balancer-tools/src/app/apr/round/(components)/MoreFiltersButton.tsx

```diff
@@ -0,0 +1,274 @@
+"use client";
+
+import { capitalize, Network } from "@bleu-balancer-tools/utils";
+import * as Accordion from "@radix-ui/react-accordion";
+import { MixerHorizontalIcon } from "@radix-ui/react-icons";
+import * as Popover from "@radix-ui/react-popover";
+import { usePathname, useRouter, useSearchParams } from "next/navigation";
+import { ChangeEvent, useCallback, useEffect, useState } from "react";
+
+import {
+  AccordionContent,
+  AccordionItem,
+  AccordionTrigger,
+} from "#/components/Accordion";
+import { Badge } from "#/components/Badge";
+import { Checkbox } from "#/components/Checkbox";
+import { BaseInput } from "#/components/Input";
+
+import { PoolTypeEnum } from "../../(utils)/calculatePoolStats";
+import { INITIAL_MIN_TVL } from "../../(utils)/getFilteredApiUrl";
+
+const networksOnBalancer = {
+  "1": Network.Ethereum,
+  "137": Network.Polygon,
+  "42161": Network.Arbitrum,
+  "100": Network.Gnosis,
+  "1101": Network.PolygonZKEVM,
+};
```

Comment:
This already exists on utils `index.ts` const `networkIdEnumMap`

Reply:
Yeah, I pretty much copied that, but I removed the networks that aren't available on Balancer's dropdown. Would you prefer me to use networkIdEnumMap's?

Reply:
you can use the `networkIdEnumMap` and do something like this:

```
const filteredKeys = [
  String(NetworkChainId.ETHEREUM),
  String(NetworkChainId.POLYGON),
  String(NetworkChainId.ARBITRUM),
  String(NetworkChainId.GNOSIS),
  String(NetworkChainId.POLYGONZKEVM),
];

const networksOnBalancerTest = Object.fromEntries(
  Object.entries(networkIdEnumMap).filter(([key]) => filteredKeys.includes(key))
);
```

and this is the same as the object you created:

```
{
  networksOnBalancer: {
    '1': 'ethereum',
    '100': 'gnosis',
    '137': 'polygon',
    '1101': 'polygon-zkevm',
    '42161': 'arbitrum'
  },
  networksOnBalancerTest: {
    '1': 'ethereum',
    '100': 'gnosis',
    '137': 'polygon',
    '1101': 'polygon-zkevm',
    '42161': 'arbitrum'
  }
}
```